### ISSUE TITLE
cent6-x64-4 ip address incorrect

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#8457]
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_Nightly/261
cent6-x64-4
`Exception creating connection to: 192.168.10.206`

**Comment by pshipton** [comment]
@jdekonin

**Comment by jdekonin** [comment]
The ip on this system changed yesterday to 192.168.10.250 but that was closer to 4-5pm, 6+ hours before this test ran.  We connect to the machine via ip, maybe the DNS wasn't updated to that point??  Can we run a grinder targeting this machine?

**Comment by pshipton** [comment]
Trying https://ci.eclipse.org/openj9/view/Test/job/Grinder/623/

**Comment by pshipton** [comment]
@jdekonin still failing the same way `Exception creating connection to: 192.168.10.206`

**Comment by pshipton** [comment]
cent6-x64-3 also has a problem. https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_x86-64_linux_xl_Nightly/263/ timed out running JvmCpuMonitorMXBean API tests which I believe is an IP address problem as per #7820.

**Comment by pshipton** [comment]
I've taken the two machines offline until this can be resolved.

**Comment by jdekonin** [comment]
@rajdeep-singh any thoughts?

**Comment by rajdeepsingh1** [comment]
@jdekonin these were built off of cent6-x64-1 - I've noticed that old IPs come up in Jenkins at times for only some machines. cent6-x64-2 was cloned the exact same way but seems to be ok.

I've modified the local /etc/hosts on the machines themselves and gave them a reboot just now which usually resolves the issue (as seen in previous cases such as ub16x64j92 I believe). 

@pshipton can these be enabled and tested? Thanks.
